### PR TITLE
i18n(es): Update remaining files

### PR DESCRIPTION
**Pull request by agustinmulet**
#### What kind of changes does this PR include?

- New or updated content

#### Description

Puts the remaining files up to date, in spanish:

- guides/typescript.md
- install/auto.md
- install/manual.md
- integrations/integrations.md
- reference/api-reference.md
- reference/configuration-reference.md

### CODE REVIEW

**Review by kevinzunigacuellar — COMMENTED** [review]
amazing work! I left a few comments 🚀

File: src/pages/es/guides/deploy.md [hunk 1]
```diff
@@ -79,6 +79,6 @@ De forma predeterminada, la salida de compilación se colocará en `dist/`. Esta
 Antes de implementar su sitio Astro con [SSR (representación del lado del servidor)](/es/guides/server-side-rendering/) habilitado, asegúrese de tener:
 
     - instaló el [adaptador apropiado](/es/guides/server-side-rendering/#habilitando-ssr-en-su-proyecto) a las dependencias de su proyecto
-    - [agregó el adaptador](/es/reference/configuration-reference/#integrations) a la importación y exportación predeterminada de su archivo `astro.config.mjs`
+    - [agregó el adaptador](/es/reference/configuration-reference/#integraciones) a la importación y exportación predeterminada de su archivo `astro.config.mjs`
```
[the remark this answers is not in the document]
Comment (agustinmulet) [reply src/pages/es/guides/deploy.md:82]: I'll add this changes but this file needs a complete review, it has some serious redaction issues

File: src/pages/es/guides/typescript.md [hunk 2]
```diff
@@ -82,19 +113,83 @@ const { greeting = 'Hello', name } = Astro.props
...
+/** @jsxImportSource solid-js */
+```
+
+### Componentes de Vue son erróneamente tipados al instalar el paquete `@types/react`
+
+Las definiciones de tipos del paquete `@types/react` son declaradas de forma global y por lo tanto son usadas para chequear tipos en los archivos `.vue` al usar [Volar](https://github.com/johnsoncodehk/volar).
+
+**Estado**: Comportamiento esperado.
+
+**Solución**: Aún no hay una forma confiable de solucionar este error, sin embargo algunas soluciones y discusiones pueden encontrarse en [esta discusión en GitHub](https://github.com/johnsoncodehk/volar/discussions/592).
```
Comment (fredoist) [inline src/pages/es/guides/typescript.md:195]:
```suggestion
**Solución**: Aún no hay una forma confiable de solucionar este error, sin embargo, algunas soluciones y discusiones pueden encontrarse en [esta discusión en GitHub](https://github.com/johnsoncodehk/volar/discussions/592).
```

File: src/pages/es/reference/api-reference.md [hunk 3]
```diff
@@ -34,10 +34,17 @@ const posts = await Astro.glob('../pages/post/*.md'); // devuelve un array de ar
...
+
 :::note
-`Astro.glob()` es un wrapper de [`import.meta.glob()`](https://vitejs.dev/guide/features.html#glob-import) de Vite, por lo que no puede aceptar variables que no sean estáticas. Consulte [la guía de solución de problemas](/es/guides/troubleshooting/#valores-compatibles) para obtener una solución alternativa.
-:::
+También puedes utilizar `import.meta.glob()` directamente en tu proyecto de Astro. Es posible que quieras hacer esto cuando:
+- Necesitas esta característica en un archivo que no sea `.astro`, como una ruta API. `Astro.glob()` solamente está disponible en archivos `.astro`, mientras que `import.meta.glob()` es accesible en todo el proyecto.
+- No necesitas cargar cada archivo de inmediatamente. `import.meta.glob()` puede devolver funciones que importan el contenido del archivo, en vez de devolver el contenido en sí.
```
Comment (fredoist) [inline src/pages/es/reference/api-reference.md:42]:
```suggestion
- No necesitas cargar cada archivo de inmediato. `import.meta.glob()` puede devolver funciones que importan el contenido del archivo, en vez de devolver el contenido en sí.
```

**Review by fredoist — CHANGES_REQUESTED** [review]
Una ultima corrección.

**Comment by kevinzunigacuellar** [comment]
✅ ready to merge

**Review by fredoist — APPROVED** [review]

**Comment by Yan-Thomas** [comment]
Good job everyone! I'll merge this PR once you have resolved the merge conflicts and got a final LGTM so I know everything is in place, thank you!

**Comment by fredoist** [comment]
Great work @agustinmulet 🤝

LGTM ✅

**Comment by Yan-Thomas** [comment]
Thanks again, everyone! Really an amazing job y'all are doing here 💜

**Pull request merged**